python/code_challenges/tree_breadth_first.py

```diff
@@ -42,3 +42,25 @@ def breadth_first(tree):
             q.enqueue(front.right)
 
     return values
+
+
+
+def fizz_buzz_tree(k_ary_tree):
+    breadth_first_order = breadth_first(k_ary_tree)
+    result = []
+    for node in breadth_first_order:
+        if node % 3 == 0 and node % 5 == 0:
```

Comment:
I'm sure this works fine but it would be better to write a separate function that implements the code here and then run it against node.value.